config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml

```diff
@@ -0,0 +1,67 @@
+periodics:
+- name: ci-kubernetes-e2e-kubeadm-kind-master
+  interval: 1h
+  labels:
+    preset-service-account: "true"
+    preset-bazel-scratch-dir: "true"
+    preset-bazel-remote-cache-enabled: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190205-d83780367-master
+      env:
+      # skip serial tests and run with --ginkgo-parallel
+      - name: "PARALLEL"
+        value: "true"
+      args:
+      - "--job=$(JOB_NAME)"
+      - "--root=/go/src"
+      - "--repo=k8s.io/kubernetes=master"
+      - "--repo=k8s.io/kubeadm=master"
+      - "--repo=sigs.k8s.io/kind=master"
+      - "--service-account=/etc/service-account/service-account.json"
+      - "--upload=gs://kubernetes-jenkins/logs"
+      - "--scenario=kubernetes_e2e"
+      - "--timeout=60"
+      - "--"
+      - "--cluster="
+      - "--build=bazel"
+      - "--provider=skeleton"
+      - "--deployment=kind"
+      - "--kind-binary-version=latest"
+      - "--kind-config./../../k8s.io/kubeadm/tests/e2e/kind/single-cp/single-cp.yaml"
+      - "--test-cmd=./../../k8s.io/kubeadm/tests/e2e/kind/test-cmd-basic.sh"
+      - "--timeout=30m"
+      # we need privileged mode in order to do docker in docker
+      securityContext:
+        privileged: true
+      # kind needs /lib/modules and cgroups from the host
+      volumeMounts:
+      - mountPath: /lib/modules
+        name: modules
+        readOnly: true
+      - mountPath: /sys/fs/cgroup
+        name: cgroup
+      resources:
```

Comment:
eventually what we should probably do is build & push a CI image periodically and then have N jobs consume the latest image, but that will come later.